modules/instrument_manager/jsx/instrumentManagerIndex.js

```diff
@@ -145,8 +145,8 @@ class InstrumentManagerIndex extends Component {
             You do not have access to this page.
           </div>
         );
-      } else if (this.state.data.writable) {
-        let url = loris.BaseURL.concat('/instrument_manager/?format=json');
+      } else if (this.state.data.fieldOptions.writable) {
```

Comment:
I think if you revert these changes it should work. `?format=json` still works when I access it manually and looks like it still has the correct data..